api/src/main/java/org/pmiops/workbench/api/DataSetController.java

```diff
@@ -228,12 +233,12 @@ public DomainValuePair apply(DataSetValues dataSetValue) {
       queryJobConfiguration = queryJobConfiguration.toBuilder().setQuery(query).build();
 
       TableResult queryResponse = bigQueryService.executeQuery(bigQueryService
-          .filterBigQueryConfig(queryJobConfiguration));
+          .filterBigQueryConfig(queryJobConfiguration), 42000l);
```

Comment:
Curious why 42000, mind adding a comment or something?

Reply:
Have updated it to a minute, Select All can sometimes take up to a minute to get the result. I have added a comment however i would want to change the logic to add retries i.e if it time out call the query again with less number of rows.